code/game/gamemodes/cult/cult_items.dm

```diff
@@ -65,7 +65,7 @@
 	icon_state = "cultrobes"
 	item_state = "cultrobes"
 	flags = FPRINT
-	allowed = list(/obj/item/weapon/tome,/obj/item/weapon/melee/cultblade)
```

Comment:
You gotta rename all the item types in this files to "<something>_legacy" since I'm working most of them in "bloodcult_items.dmi"

Reply:
Alright, almost all of them but the blade are already btw

Reply:
and the robes, hoods, armor, etc

Reply:
This file is commented out you silly billy